python/pacsign/PACSign.md

```diff
@@ -0,0 +1,166 @@
+# PACSign #
+
+## SYNOPSIS ##
+`python PACSign.py [-h] {FIM,SR,BBS,BMC,BMC_FW,AFU,PR,GBS} ...`
+
+`python PACSign.py <CMD>  [-h] -t {UPDATE,CANCEL,RK_256,RK_384} -H HSM_MANAGER
+                          [-C HSM_CONFIG] [-r ROOT_KEY] [-k CODE_SIGNING_KEY]
+                          [-d CSK_ID] [-i INPUT_FILE] [-o OUTPUT_FILE] [-y] [-v]`
+
+## DESCRIPTION ##
+The `PACSign` utility inserts authentication markers into bitstreams targeted for the following programmable
+acceleration cards (PACs):
+* Intel&reg; PAC with Intel Arria&reg; 10 GX FPGA
+* Intel FPGA PAC D5005
+* Intel PAC N3000
+`PACSign` uses a root key and an optional code signing key to digitally sign the  
+bitstreams to validate their origin. The PACs will not program bitstreams without proper authentication.
+
+The current PACs only support elliptical curve keys with the curve type `secp256r1` or `prime256v1`.
+The `PACSign` command supports hardware security modules (HSMs) for both `OpenSSL` and `PKCS #11`.
```

Comment:
@r-rojo a snippet can be added here to modify `PKCS11_config.json` file's three dummy fields if you want to use `PKCS #11`